cpp/include/cudf/ast/expressions.hpp

```diff
@@ -46,27 +46,35 @@ struct expression {
  */
 enum class ast_operator {
   // Binary operators
-  ADD,            ///< operator +
-  SUB,            ///< operator -
-  MUL,            ///< operator *
-  DIV,            ///< operator / using common type of lhs and rhs
-  TRUE_DIV,       ///< operator / after promoting type to floating point
-  FLOOR_DIV,      ///< operator / after promoting to 64 bit floating point and then
-                  ///< flooring the result
-  MOD,            ///< operator %
-  PYMOD,          ///< operator % but following python's sign rules for negatives
-  POW,            ///< lhs ^ rhs
-  EQUAL,          ///< operator ==
-  NOT_EQUAL,      ///< operator !=
-  LESS,           ///< operator <
-  GREATER,        ///< operator >
-  LESS_EQUAL,     ///< operator <=
-  GREATER_EQUAL,  ///< operator >=
-  BITWISE_AND,    ///< operator &
-  BITWISE_OR,     ///< operator |
-  BITWISE_XOR,    ///< operator ^
-  LOGICAL_AND,    ///< operator &&
-  LOGICAL_OR,     ///< operator ||
+  ADD,               ///< operator +
+  SUB,               ///< operator -
+  MUL,               ///< operator *
+  DIV,               ///< operator / using common type of lhs and rhs
+  TRUE_DIV,          ///< operator / after promoting type to floating point
+  FLOOR_DIV,         ///< operator / after promoting to 64 bit floating point and then
+                     ///< flooring the result
+  MOD,               ///< operator %
+  PYMOD,             ///< operator % but following python's sign rules for negatives
```

Comment:
This operator's comment doesn't match the updates for Java bindings in 6971f52fc46205278eb2fdfa22573d51bec65e6f. It would be good to make these match. (Also capitalize the word "Python" in the Java code's comment.)
```suggestion
  PYMOD,             ///< operator % using Python's sign rules for negatives
```